spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderAutoConfigurationPropertiesTest.java

```diff
@@ -42,8 +36,15 @@
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 import org.springframework.util.ReflectionUtils;
 
+import java.lang.reflect.Field;
```

Comment:
Wrong import order - see https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style#import-statements - can be fixed on merge